### DISCUSSION TITLE
xstate v5 beta parent machine not running or re-starting spawned actors in Nextjs

### DISCUSSION THREAD

**Discussion by murrowblue22** [statelyai/xstate#4147] [Q&A]
I'm using xstate v5, and i want to be able to restore my parent machine that has spawned machines from local storage, but it does not work.  In my app i have a parent machine that just stores child machines for my app that are lazy loaded into and array when the root path of a section of my web app is loaded. 

the main code i'm using is 
```ts
const modelMachineActorRef = useActorRef(machine)
const initialPersistedState = modelMachineActorRef.getPersistedState!()
localStorage.setItem('pam_service_state', JSON.stringify(initialPersistedState))
```
then later i use the following code to restore the machine, obviously i'm not putting all step just the key parts 
```ts
const [state, send, pam_service] = useActor(machine, {
        state: JSON.parse(localStorage.getItem('pam_service_state')!)
})
```

however the restored machine does not run, nor does it start spawned child actors.  My question is this a known bug in xstate v5, or am i trying to use functionality that is just not present

**Comment by davidkpiano** [comment]
Can you share a runnable code example that demonstrates the problem? Restoring persisted actors should work, so it may be a problem with your specific use-case or implementation.
